From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dreeam <61569423+Dreeam-qwq@users.noreply.github.com>
Date: Wed, 31 Jul 2024 22:05:21 +0800
Subject: [PATCH] Leaf Bootstrap

Removed since Leaf 1.21.4, useless

org.bukkit.craftbukkit.Main#main -> LeafBootstrap -> PaperBootstrap -> ...

diff --git a/src/main/java/org/bukkit/craftbukkit/Main.java b/src/main/java/org/bukkit/craftbukkit/Main.java
index bf5343b0847e9f57ffbc7f33714ae6ca62f14332..70d093f71cca569d9da3ae82e738f8f3069f405e 100644
--- a/src/main/java/org/bukkit/craftbukkit/Main.java
+++ b/src/main/java/org/bukkit/craftbukkit/Main.java
@@ -278,7 +278,8 @@ public class Main {
                 System.setProperty("jdk.console", "java.base"); // Paper - revert default console provider back to java.base so we can have our own jline
                 //System.out.println("Loading libraries, please wait...");
                 //net.minecraft.server.Main.main(options);
-                io.papermc.paper.PaperBootstrap.boot(options);
+                //io.papermc.paper.PaperBootstrap.boot(options); // Leaf - Leaf Boostrap - diff on change
+                org.dreeam.leaf.LeafBootstrap.boot(options); // Leaf - Leaf Boostrap
             } catch (Throwable t) {
                 t.printStackTrace();
             }
